src/sagas/wallet.js

```diff
@@ -266,6 +268,11 @@ export function* startWallet(action) {
       // takeLatest will stop running the generator if a new START_WALLET_REQUESTED
       // action is dispatched, but returning so the code is clearer
       return;
+    } else {
+      console.error(e);
+      // Return to locked screen when the wallet fails to start
+      LOCAL_STORE.lock();
+      routerHistory.push('/locked/');
```

Comment:
Sometimes the wallet would be stuck on loading addresses, the error I got was that the configured network and server network did not match, it was raised during the start, but we already have a `try..catch` it just didn't handle the error, so the sagas would be waiting on a wallet that was not started.

The solution here is just to go back in the locked screen, but we can also add an error message in the future.
The reason I was having this error was because the change server error this PR solves would change the server but did not change the network so I do not expect to have this error again but i'm leaving this to make debugging future errors of this nature easier.